src/azure-cli/azure/cli/command_modules/storage/tests/latest/test_storage_azcopy_scenarios.py

```diff
@@ -10,6 +10,8 @@
 
 
 class StorageAzcopyTests(StorageScenarioMixin, LiveScenarioTest):
```

Comment:
Do we need to write test_scenario for hybrid_2018_03_01 or hybrid_2019_03_01?

Reply:
This command only supports the latest profile. No plan to support other profiles.